Assets/Mirror/Runtime/NetworkServer.cs

```diff
@@ -218,7 +218,7 @@ internal void ActivateHostScene()
                 {
                     if (LogFilter.Debug) Debug.Log("ActivateHostScene " + identity.netId + " " + identity);
 
-                    identity.OnStartClient();
+                    identity.OnStartClient.Invoke();
```

Comment:
should this be identity.StartClient() instead?

Reply:
It would work but then it would not call the other listeners for that identity for this that event.

Reply:
Resolved in recent commits. Please re-review.